crates/pop-cli/src/commands/build/mod.rs

```diff
@@ -12,18 +19,72 @@ pub(crate) mod parachain;
 #[command(args_conflicts_with_subcommands = true)]
 pub(crate) struct BuildArgs {
 	#[command(subcommand)]
-	pub command: Command,
+	pub command: Option<Command>,
+	/// Directory path for your project [default: current directory]
+	#[arg(long)]
+	pub(crate) path: Option<PathBuf>,
+	/// The package to be built.
+	#[arg(short = 'p', long)]
+	pub(crate) package: Option<String>,
+	/// For production, always build in release mode to exclude debug features.
+	#[clap(short, long)]
+	pub(crate) release: bool,
 }
 
 /// Build a parachain or smart contract.
```

Comment:
Not sure if it makes sense to add something like: "Build a parachain, smart contract, or any Rust project." Technically, you can even build `pop-cli `, pallets and packages with this command. Up to you.

Reply:
Will change it simply to "Build a project", only as how it currently reads doesnt take features into account and having different comments for the highlevel command based on features seems like overkill.

Reply:
Looking at it now, a lot of the comments suffer the same fate, so maybe I should address it properly so we have a reference to update the rest.

Reply:
Addressed in latest commit.